README.md

```diff
@@ -211,16 +211,24 @@ You can see the build statuses at
 [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/home) is
 required to run the frontend and backend test suites.
 
-To execute all of the test specs, run the `build.sh` script at the root of the Solidus project:
+To execute all of the test specs, run the `bin/build` script at the root of the Solidus project:
+
+```bash
+createuser --superuser --echo postgres # only the first time
+bin/build
+```
+
+The `bin/build` script runs using PostgreSQL by default, but it can be overridden by setting the DB environment variable to `DB=sqlite` or `DB=mysql`. For example:
 
 ```bash
-bash build.sh
+env DB=mysql bin/build
 ```
 
-The `build.sh` script runs using PostgreSQL by default, but it can be overridden by setting the DB environment variable to `DB=sqlite` or `DB=mysql`. For example:
+If the command fails with MySQL related errors you can try creating a user with this command:
 
 ```bash
-DB=mysql bash build.sh
+# Creates a user with the same name as the current user and no restrictions.
+mysql -uroot -e"CREATE USER '$USER'@'localhost'; GRANT ALL PRIVILEGES ON * . * TO '$USER'@'localhost';"
```

Comment:
I believe you are missing a space after `mysql -uroot -e`

Reply:
It's fine, for single letter options you're allowed to keep the argument attached to the option. But I'll change them to their expanded version because it's more understandable for the casual reader.